script/toy/MangroveJs.s.sol

```diff
@@ -26,21 +27,23 @@ interact with.  b) For any additional deployments needed, those files should be
 hosted in mangrove.js.*/
 
 contract MangroveJsDeploy is Deployer {
-  TestToken tokenA;
-  TestToken tokenB;
-  IERC20 dai;
-  IERC20 usdc;
-  IERC20 weth;
-  SimpleTestMaker simpleTestMaker;
-  MangroveOrder mgo;
+  TestToken public tokenA;
+  TestToken public tokenB;
+  IERC20 public dai;
+  IERC20 public usdc;
+  IERC20 public weth;
+  SimpleTestMaker public simpleTestMaker;
+  MangroveOrder public mgo;
 
   function run() public {
     innerRun({chief: broadcaster(), gasprice: 1, gasmax: 2_000_000, gasbot: broadcaster()});
     outputDeployment();
   }
 
   function innerRun(address chief, uint gasprice, uint gasmax, address gasbot) public {
-    fork.set("MgvGovernance", chief);
+    if (chief != broadcaster()) {
+      console.log("Warning: chief is not the broadcaster; deployed contracts will not obey their orders.");
+    }
```

Comment:
Maybe failing fast is better? This way it will be clear, that a core assumption is not met. Or, as you suggest in the PR description, to simply remove the `chief` argument to prevent the issue altogether.